DFE-Digital/ecf-engage-and-learn
# Cpdel 708 guidance speed bump

## Ticket and context

Ticket: 

[708](https://dfedigital.atlassian.net/jira/software/projects/CPDEL/boards/89?selectedIssue=CPDEL-708)
[765](https://dfedigital.atlassian.net/jira/software/projects/CPDEL/boards/89?selectedIssue=CPDEL-765)

708 is the bulk of the change implemented. 765 is just the addition of a button on guidance pages and rewording a link. 

## Code review

### Is there anything weird that code reviewer should know?
I found raising an error when no radio is selected a pain. Passing true or false as a value meant that a radio button was preselected, which we don't want. 

Naming, as always, is a pain so welcome any thoughts on naming appropriately. 

### Review Checks

## Product review

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. login is a mentor or ect 
3. Click start now on the dashboard. 
4. Go through the multiple scenarios set out in the acceptance criteria. 
5. You should only see the page once for each user type so you'll need to log in to another example account to try again.
